src/ucp/wireup/wireup_cm.c

```diff
@@ -118,6 +118,18 @@ static int ucp_cm_client_try_fallback_cms(ucp_ep_h ep)
     return 1;
 }
 
+static ucp_rsc_index_t
+ucp_cm_ep_tl_bitmap_get_dev_idx(ucp_worker_h worker, uint64_t tl_bitmap)
+{   
+    ucp_rsc_index_t rsc_index;
+
+    ucs_assert(tl_bitmap != 0);
+    rsc_index = ucs_ffs64_safe(tl_bitmap);
+    ucs_assert(rsc_index != UCP_MAX_RESOURCES);
```

Comment:
this also duplicates `ucs_assert(tl_bitmap != 0);` in some way, because this assert checks at least one bit is set 
wdyt?

Reply:
AFAIR this assert is needed to suppress coverity

Reply:
> this also duplicates `ucs_assert(tl_bitmap != 0);` in some way, because this assert checks at least one bit is set
> wdyt?

ok, let's then drop the `ucs_assert(tl_bitmap != 0)`?

Reply:
i do not think coverity is related, this assert was not present before

Reply:
> > this also duplicates `ucs_assert(tl_bitmap != 0);` in some way, because this assert checks at least one bit is set
> > wdyt?
> 
> ok, let's then drop the `ucs_assert(tl_bitmap != 0)`?

up to you what assert to remove, `ucs_assert(tl_bitmap != 0)` looks more clear to me